common/src/main/java/com/wynntils/models/ingredients/IngredientModel.java

```diff
@@ -17,40 +16,23 @@
 import java.util.Optional;
 import java.util.stream.Stream;
 import net.minecraft.ChatFormatting;
-import net.neoforged.bus.api.SubscribeEvent;
 
 public class IngredientModel extends Model {
     private static final Map<ChatFormatting, Integer> TIER_COLOR_CODES = Map.of(
             ChatFormatting.DARK_GRAY, 0,
             ChatFormatting.YELLOW, 1,
             ChatFormatting.LIGHT_PURPLE, 2,
             ChatFormatting.AQUA, 3);
+
     private final IngredientInfoRegistry ingredientInfoRegistry = new IngredientInfoRegistry();
 
     public IngredientModel(WynnItemModel wynnItem) {
```

Comment:
I'm guessing the dependency in wynnItem can be removed now? My reading of this code was that it was added either when we loaded the ingredient DB in the constructor, or as a safe guard against a race condition when it was made to be asynchronously loaded.

Reply:
That's likely correct. Or maybe when our "dependency" requirement was more strict. I'll remove it and check other cases as well.